### ISSUE TITLE
Being careful how you write code - baby steps

### ISSUE THREAD

**Issue by einarc** [nodeschool/discussions#459]
Even though you might follow ALL the suggestions the tool gives you because of the way it compares code you might trigger a fail. 

For example this code Yields a fail:

```
console.log(process.argv);
var sum=0
for (i=2;i<process.argv.length;i++)
{
        sum=sum+Number(process.argv[i]);
}

console.log(sum);
```

ubuntu@ip-10-168-221-58:~/node/baby$ learnyounode verify add.js

Your submission results compared to the expected:

```
             ACTUAL                                  EXPECTED                 
```

────────────────────────────────────────────────────────────────────────────────

   "[ '/usr/local/bin/node',"          !=    "88"  
   "  '/home/ubuntu/node/baby/add.js'," !=    ""  
   "  '5',"                            !=  
   "  '46' ]"                          !=  
   "88"                                !=  
   ""                                  !=                                       

────────────────────────────────────────────────────────────────────────────────

✗ Submission results match expected
# FAIL

Your solution to BABY STEPS didn't pass. Try again!

But THIS code passes:

```
var sum=0
for (i=2;i<process.argv.length;i++)
{
        sum=sum+Number(process.argv[i]);
}

console.log(sum);
```

ubuntu@ip-10-168-221-58:~/node/baby$ vi add.js 
ubuntu@ip-10-168-221-58:~/node/baby$ learnyounode verify add.js

Your submission results compared to the expected:

```
             ACTUAL                                  EXPECTED                 
```

────────────────────────────────────────────────────────────────────────────────

   "69"                                ==    "69"  
   ""                                  ==    ""                                 

────────────────────────────────────────────────────────────────────────────────

✓ Submission results match expected
# PASS

Your solution to BABY STEPS passed!

Here's the official solution in case you want to compare notes:

────────────────────────────────────────────────────────────────────────────────
    var result = 0

```
for (var i = 2; i < process.argv.length; i++)
  result += Number(process.argv[i])

console.log(result)
```

────────────────────────────────────────────────────────────────────────────────

You have 11 challenges left.

So in summary, even though the logic is the same just because of an extra line the test will fail your code. This doesn't leave much hope for more complicated stuff.
